src/cmd/build.rs

```diff
@@ -5,8 +5,9 @@ use site::Site;
 
 use console;
 
-pub fn build(config_file: &str, base_url: Option<&str>) -> Result<()> {
+pub fn build(config_file: &str, base_url: Option<&str>, public: &str) -> Result<()> {
```

Comment:
I would name `public` `output_dir` instead, public is not very explicit in the code